packages/upload/src/vaadin-upload.js

```diff
@@ -902,6 +898,26 @@ class Upload extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElement))
     }
   }
 
+  /** @private */
+  _getAcceptRegex() {
+    if (!this.accept) {
+      return;
+    }
+    const processedAccept = this.accept.split(',').map((token) => {
```

Comment:
```suggestion
    const processedTokens = this.accept.split(',').map((token) => {
```

Reply:
Included this change in the latest commit.